src/NzbDrone.Core/Organizer/FileNameBuilder.cs

```diff
@@ -698,6 +698,15 @@ private void AddCustomFormats(Dictionary<string, Func<TokenMatch, string>> token
             }
 
             tokenHandlers["{Custom Formats}"] = m => string.Join(" ", customFormats.Where(x => x.IncludeCustomFormatWhenRenaming));
+            tokenHandlers["{Custom Format}"] = m =>
+            {
+                if (m.CustomFormat.IsNullOrWhiteSpace())
+                {
+                    return string.Empty;
+                }
+
+                return string.Join(" ", customFormats.Where(x => x.IncludeCustomFormatWhenRenaming && x.Name == m.CustomFormat));
```

Comment:
If Custom Format names are unique, then this could be simplified to just
`customFormats.Where(x => x.IncludeCustomFormatWhenRenaming && x.Name == m.CustomFormat).FirstOrDefault()?.ToString() ?? string.Empty`

Reply:
They are unique. https://github.com/Sonarr/Sonarr/blob/develop/src/Sonarr.Api.V3/CustomFormats/CustomFormatController.cs#L28-L29